doc/man1/openssl-x509.pod.in

```diff
@@ -348,10 +348,10 @@ can thus behave like a "mini CA".
 
 =over 4
 
-=item B<-signkey> I<filename>
+=item B<-signkey> I<arg>
 
 This option causes the input file to be self signed using the supplied
-private key.
+private key or engine.
```

Comment:
The text should mention that what it takes depens on `-keyform`

Reply:
Okay. I mentioned keyform additionally. #11085 corrects the keyform description to also mention ENGINE.